src/main/java/io/r2dbc/pool/PoolMetrics.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.r2dbc.pool;
+
+import reactor.pool.Pool;
+
+/**
+ * An object that can be used to get live information about a {@link ConnectionPool}, suitable
+ * for gauge metrics.
+ * <p>
+ * {@code getXxx} methods are configuration accessors, their values that won't change over time, whereas other methods can be used as gauges to introspect the current state of the pool.
+ *
+ * @author Mark Paluch
+ */
+public interface PoolMetrics {
```

Comment:
I currently do not understand why this file is marked as new as this was already merged (https://github.com/r2dbc/r2dbc-pool/blob/master/src/main/java/io/r2dbc/pool/PoolMetrics.java).